src/harness/reference_models/dpa/move_list.py

```diff
@@ -182,7 +171,8 @@ def findGrantsInsideNeighborhood(grants, constraint,
         if grant.cbsd_category == 'A':
             if dpa_type is not DpaType.OUT_OF_BAND:
                 point_cbsd = SPoint(lon_cbsd, lat_cbsd)
-                if dist_km <= CAT_A_NBRHD_DIST and polygon_ex_zone.contains(point_cbsd):
+                if (dist_km <= CAT_A_NBRHD_DIST and
+                    (polygon_ex_zone is None or polygon_ex_zone.contains(point_cbsd))):
                     cbsd_in_nbrhd = True
         elif grant.cbsd_category == 'B':
```

Comment:
I think there is an issue with comparing "dist_km" with "catb_neighbor_dist" and then with "CAT_B_NBRHD_FOR_HT" (300 km). It is ok if "catb_neighbor_dist" is equal to "CAT_B_NBRHD_DIST_DEFAULT" (200 km), but what if catb_neighbor_dist > CAT_B_NBRHD_FOR_HT? 
Do you think we should keep the original code that implemented the current R2-SGN-24.a.ii. if catb_neighbor_dist = None? If catb_neighbot_dist is not None, we only compare dist_km with catb_neighbor_dist?